spmd/tensor/dispatch.py

```diff
@@ -132,6 +132,9 @@ def operator_dispatch(
     args_schema = tree_map(unwrap_schema, args)
     kwargs_schema = tree_map(unwrap_schema, kwargs)
 
+    # TODO(anj): Check if the input to ops are all DTs with placements specs.
```

Comment:
this check has already been added here https://github.com/pytorch/tau/blob/main/spmd/tensor/api.py#L217